Druid/DRCogitDispatchTableBuilder.class.st

```diff
@@ -17,7 +17,87 @@ DRCogitDispatchTableBuilder >> arrayAccessorGlobalName [
 	^ #CArrayAccessor
 ]
 
-{ #category : #'accessing - building' }
+{ #category : #'accessing - bytecodes' }
+DRCogitDispatchTableBuilder >> buildBytecodeTableVariableNode [
```

Comment:
So the builder does not build the dispatch table, right? The builder is more like a code generator for the dispatch table?